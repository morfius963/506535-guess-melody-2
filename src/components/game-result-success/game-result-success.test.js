import React from "react";
import renderer from "react-test-renderer";
import GameResultSuccess from "./game-result-success.jsx";

jest.mock(`react-router-dom`, () => ({
  useHistory: () => ({
    push: jest.fn(),
  }),
}));

describe(`snapshot test`, () => {
  it(`Component correctly renders`, () => {
    const props = {
      time: 50000,
      mistakes: 1,
      restartGame: jest.fn()
    };

    const tree = renderer
      .create(<GameResultSuccess {...props} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
