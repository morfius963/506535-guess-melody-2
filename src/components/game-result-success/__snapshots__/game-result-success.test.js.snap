// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test Component correctly renders 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height={83}
      src="img/melody-logo.png"
      width={186}
    />
  </div>
  <h2
    className="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="result__total"
  >
    Вы играли 
    04:10
     и набрали 
     баллов (
     быстрых ответов), совершив 
    1
     ошибки
  </p>
</section>
`;
